spec/factories/ticket_purchases.rb

```diff
@@ -4,5 +4,6 @@
     conference
     ticket
     quantity 10
+    paid false
```

Comment:
Yes I think it would be useful to have a sub factory here for paid ticket_purchases.